### PR TITLE
test: add tests for utils 

### PR DESCRIPTION
**Pull request by murrayee**
Added test coverage for utils `bytes.ts`

> toHex
> fromHex
> toHexString

Current coverage： 87.5%

### CODE REVIEW

**Review by dapplion — COMMENTED** [review]
Thanks! Can you write the tests in a `TestCase[]` pattern to reduce boilerplate and make it easier to audit?

See example

https://github.com/ChainSafe/lodestar/blob/fa30bcf10086292883ef086202474ac4df67ffe1/packages/utils/test/unit/retry.test.ts#L17

**Comment by murrayee** [comment]
> Thanks! Can you write the tests in a `TestCase[]` pattern to reduce boilerplate and make it easier to audit?
> 
> See example
> 
> https://github.com/ChainSafe/lodestar/blob/fa30bcf10086292883ef086202474ac4df67ffe1/packages/utils/test/unit/retry.test.ts#L17

Thanks！ I've never written tests like this before, So perfect ！！

**Review by dapplion — DISMISSED** [review]
Looks great! Thank you for the contribution

File: packages/utils/test/unit/bytes.test.ts [hunk 1]
```diff
@@ -47,3 +47,54 @@ describe("bytesToInt", () => {
...
+    },
+    {
+      input: "48656c6c6f2c20576f726c6421",
+      output: new Uint8Array([72, 101, 108, 108, 111, 44, 32, 87, 111, 114, 108, 100, 33]),
+    },
+    {input: "0x", output: new Uint8Array([])},
+  ];
+
+  for (const {input, output} of testCases) {
+    it(`should convert  hex string ${input} to Uint8Array`, () => {
```
Comment (nflaig) [inline packages/utils/test/unit/bytes.test.ts]:
```suggestion
    it(`should convert hex string ${input} to Uint8Array`, () => {
```

File: packages/utils/test/unit/bytes.test.ts [hunk 2]
```diff
@@ -47,3 +47,54 @@ describe("bytesToInt", () => {
...
+    {input: new Uint8Array([1, 2, 3]), output: "0x010203"},
+    {input: new Uint8Array([72, 101, 108, 108, 111]), output: "0x48656c6c6f"},
+    {input: new Uint8Array([]), output: "0x"},
+    {input: new Uint8Array([0, 0, 0, 0]), output: "0x00000000"},
+    {input: new Uint8Array([15, 255, 16, 0, 127]), output: "0x0fff10007f"},
+    {input: new Uint8Array(1000).fill(255), output: "0x" + "ff".repeat(1000)},
+  ];
+
+  for (const {input, output} of testCases) {
+    it(`should convert Uint8Array to hex string ${output.length}`, () => {
```
Comment (nflaig) [inline packages/utils/test/unit/bytes.test.ts]: printing the output length is not that informative, could reduce size of last test case and print expected hex string (`output`)

**Review by nflaig — APPROVED** [review]
lgtm, thanks for the updates!

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.13.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.13.0) :tada:

**Pull request merged**